src/generic/datavgen.cpp

```diff
@@ -1618,6 +1621,7 @@ class wxDataViewDropTarget: public wxDropTarget
         wxDataFormat format = GetMatchingPair();
         if (format == wxDF_INVALID)
             return wxDragNone;
+
```

Comment:
Please avoid whitespace-only changes combined with other, non-trivial ones (here and below).